### PR TITLE
Setup react app

### PR DESCRIPTION
**Pull request by carloszanella**
Does the initial setup. you can run it with `yarn start`

```
├── LICENSE
├── README.md
├── jest.config.js
├── package.json
├── public
│   ├── favicon.ico
│   ├── index.html
│   ├── logo192.png
│   ├── logo512.png
│   ├── manifest.json
│   └── robots.txt
├── src
│   ├── App.css
│   ├── App.test.tsx
│   ├── App.tsx
│   ├── index.css
│   ├── index.tsx
│   ├── logo.svg
│   ├── privacy
│   │   ├── threshold.test.tsx
│   │   └── threshold.tsx
│   ├── react-app-env.d.ts
│   ├── reportWebVitals.ts
│   └── setupTests.ts
├── tsconfig.json
├── wagmi.config.js
├── yarn-error.log
└── yarn.lock
```

### CODE REVIEW

**Review by dianacarvalho1 — APPROVED** [review]
👍🏼 👍🏼 👍🏼

File: package.json [hunk 1]
```diff
@@ -1,15 +1,45 @@
 {
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
   "dependencies": {
-    "typescript": "^5.0.4",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
```
Comment (kayibal) [inline package.json:11]: Should these all be dependencies? I think these should be dev dependencies...

File: package.json [hunk 2]
```diff
@@ -1,15 +1,45 @@
 {
+  "name": "my-app",
```
Comment (kayibal) [inline package.json:2]: ?

File: package.json [hunk 3]
```diff
@@ -1,15 +1,45 @@
...
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.4.2",
     "viem": "^0.3.22",
-    "wagmi": "^1.0.2"
-  },
-  "devDependencies": {
```
Comment (kayibal) [inline package.json:7]: Why did you remove this?
